runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesOverlay.java

```diff
@@ -47,30 +60,33 @@
 		setPriority(OverlayPriority.LOW);
 		this.plugin = plugin;
 		this.config = config;
+		horizontalContainer.setOrientation(PanelComponent.Orientation.HORIZONTAL);
 	}
 
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
+		horizontalContainer.getChildren().clear();
 		Map<Integer, Integer> teams = plugin.getTeams();
 		if (teams.isEmpty())
 		{
 			return null;
 		}
-		panelComponent.getChildren().clear();
-
 		for (Map.Entry<Integer, Integer> team : teams.entrySet())
 		{
-			// Only display team capes that have a count greater than the configured minimum.
-			if (team.getValue() >= config.getMinimumCapeCount())
+			// Only display team capes that have a count greater than the configured minimum AND teamcapes should be relevant to pking ( < 51)
```

Comment:
Just display all team capes and make the panel component to wrap after like 6 entries or something (it has property for it).